question/tests/qbank_columnsortorder_test.php

```diff
@@ -0,0 +1,110 @@
+<?php
+// This file is part of Moodle - http://moodle.org/
+//
+// Moodle is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// Moodle is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with Moodle.  If not, see <http://www.gnu.org/licenses/>.
+
+defined('MOODLE_INTERNAL') || die();
+use core_question\local\bank\helper;
+use core_question\local\bank\view;
+global $CFG;
+require_once($CFG->dirroot . '/question/tests/fixtures/testable_core_question_column.php');
+require_once($CFG->dirroot . '/question/classes/external.php');
+
+/**
+ * Question bank settings page class.
+ *
+ * @package    qbank_columnsortorder
+ * @copyright  2021 Catalyst IT Australia Pty Ltd
+ * @author     Ghaly Marc-Alexandre <marc-alexandreghaly@catalyst-ca.net>
+ * @license    http://www.gnu.org/copyleft/gpl.html GNU GPL v3 or later
+ */
+
+class qbank_columnsortorder_test extends advanced_testcase {
```

Comment:
1. Namespacing not used because `question/classes/external.php`  has no namespace declared, I might brake core code if I declare it, I ll need to use other classes in the file mentioned.
2. Name and classes changed for `columsortorder_test` accordingly.

Reply:
HI @marcghaly for this class: `namespace core_question;`. After this one, It's good to be merged. Thank you

Reply:
fixed